src/collection.js

```diff
@@ -494,6 +494,21 @@ class Collection {
     return this
   }
 
+  /**
+   * Returns median of the current collection
+   *
+   * @param {}
+   *
+   * @returns {Number}
+   */
+  async median () {
+    const arr = (await this.sort()).items
+    const mid = Math.floor(arr.length / 2)

```

Comment:
Without the temporary `arr` variable, you can use `this.size()` to determine the number of items:

```suggestion
    const mid = Math.floor(this.size() / 2)
```

Reply:
Done. Please Check